modules/core/src/test/java/org/apache/ignite/internal/processors/cache/mvcc/CacheMvccConfigurationValidationTest.java

```diff
@@ -322,6 +325,33 @@ public void testTransactionalSnapshotLimitations() throws Exception {
         );
     }
 
+    /**
+     * Check that node in client mode (filtered by AttributeNodeFilter) correctly works with MVCC.
+     */
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testMvccEnabledForClientMode() throws Exception {
+        String attrName = "has_cache";
+        Object attrVal = Boolean.TRUE;
+
+        final IgniteEx crd = (IgniteEx) startGrid(getTestIgniteInstanceName(0), getConfiguration());
+
+        // Do not start cache on non-affinity node.
+        CacheConfiguration ccfg = defaultCacheConfiguration()
```

Comment:
Do we really need to specify near, backups, and atomicity mode here?

Reply:
Hi, this test checks that MVCC mode is enabled, so atomicity mode is required for testing. Near must be set to null with enabled MVCC. Backups doesn't matter actually, I'll fix it

Reply:
Hi, this test checks MVCC mode so atomicity mode is required. Near set is required as it must be disabled with MVCC mode. Backups dont matter actually. Fixed it